torchtitan/models/norms.py

```diff
@@ -284,7 +284,6 @@ def backward(ctx, dy):
 
         M, N = dy.shape
         dx = torch.empty_like(x)
-        dw = torch.empty_like(weight)
```

Comment:
unused, `dw` is directly computed:
https://github.com/pytorch/torchtitan/blob/b7754d650ee486f8963949e79fe471fe8af6b11a/torchtitan/models/norms.py#L315